spec/http.md

```diff
@@ -0,0 +1,51 @@
+HTTP
+====
+
+## Resources
+
+Each EC2 instance MUST present the following (GET) resources:
+
+    [/service/healthcheck](#healthcheck)
+    [/service/gtg](#gtg)
+    [/service/dependencies](#dependencies)
+
+These resources should be presented on port 8080 (not port 80).
+
+### <a name="healthcheck">Healthcheck</a>
+
+Is the service running? Two possible responses:
+
+    200 "OK" (service is running)
```

Comment:
Yes. If it's a 200 response then you don't care about the body so you don't need to bother parsing the JSON, but if it's a 5xx then you (or rather a script) might want to extract detailed info from the body.